basilisk/gui/main_frame.py

```
@@ -364,14 +364,14 @@ def screen_capture(
 		)
 		thread.start()
 
-	def post_screen_capture(self, imagefile: ImageFile | str):
+	def post_screen_capture(self, image_file: ImageFile | str):
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)